### DISCUSSION TITLE
Inloggen als Instructeur in test omgeving

### DISCUSSION THREAD

**Discussion by StefanLeli** [Nelos-VZW/dives-2#48] [Ideas]
Is het mogelijk om als instructeur in te loggen om zo de functies direct te testen zonder dat je een instructeur moet gaan lastig vallen.
In Dives 1.0 was dit een mogelijkheid in de test omgeving.

![image](https://user-images.githubusercontent.com/112511522/188803927-95b70a47-1a6e-41a5-93c5-222a6d1e3199.png)

**Comment by ndsmyter** [comment]
We hebben in DIVES 2.0 een functie ingebouwd waarbij je zelf "test-users" kan aanmaken. Dat zijn accounts die aangemaakt worden volgens een bepaald profiel, waardoor je onmiddellijk al X aantal duiken en dergelijke gelogd hebt.

Als je mij jouw NELOS ID bezorgt, kan ik je wel toevoegen aan die groep.

Eenmaal toegevoegd zal je in het menu een optie "Test Team" bij krijgen, waar je de accounts kan aanmaken.

### Stappenplan

<img width="591" alt="image" src="https://user-images.githubusercontent.com/648669/190093133-b38bccdb-5ab9-4cdd-b38b-9e948f3c2966.png">

Selecteer het gewenste profiel:
<img width="607" alt="image" src="https://user-images.githubusercontent.com/648669/190093267-b446b113-e5c8-4278-936e-88abe9287fe7.png">

De details van de nieuwe gebruiker:
<img width="294" alt="image" src="https://user-images.githubusercontent.com/648669/190093329-5275078d-d805-418c-bdce-b05b03d80414.png">

Dit zijn dan ook de gegevens waarmee je kan inloggen. Je ziet namelijk het NELOS ID en het wachtwoord opgelijst staan.

PS: De gebruikers zijn iets langer geldig dan de melding aangeeft. Ze zijn geldig tot de volgende migratie. Op test.nelos.be is dat dus tot maandag-, woensdag- of vrijdagmorgen.

**Comment by StefanLeli** [comment]
Hey Nicolas,
Mijn Nelos ID is 32180
Alvast bedankt




Op wo 14 sep. 2022 om 09:48 schreef Nicolas De Smyter <
***@***.***>:

**Reply by ndsmyter** [reply]
Ik heb je nu toegevoegd. Zal wel pas actief zijn na de volgende migratie morgen (16/09/22).
